src/explorer/DeploymentSlotsTreeItem.ts

```diff
@@ -4,12 +4,12 @@
  *--------------------------------------------------------------------------------------------*/
 
 import WebSiteManagementClient = require('azure-arm-website');
-import { ResourceNameAvailability } from 'azure-arm-website/lib/models';
-import { NameValuePair, Site, WebAppCollection } from 'azure-arm-website/lib/models';
+import { NameValuePair, ResourceNameAvailability, Site, WebAppCollection } from 'azure-arm-website/lib/models';
 import * as path from 'path';
 import { window } from 'vscode';

```

Comment:
Could get rid of this if you got rid of line 139 window.showQuickPick and replaced it with our ext.ui

Reply:
Discovered a bug due to this comment, so I submitted a separate PR with the fix: https://github.com/Microsoft/vscode-azureappservice/pull/440